woocommerce/woocommerce-ios
# [Mobile Payments] Settings 2-8b: Swap out scanning alert with new modal

Closes #4163 

Changes:
- Replaces UIAlert displayed when scanning for readers with new modal

Unfinished:
- Matching styles in design exactly

To test:
- Open Settings > Manage Card Readers V2 while no reader is connected
- Tap on Connect button
- Ensure new modal appears
- Tap on Cancel
- Ensure alert disappears
- You should also go through the payments flow and ensure all those modals appear correct since some tweaks were made to bottom label handling in the modals

Update release notes:

![Screen Shot 2021-05-11 at 10 55 46 AM](https://user-images.githubusercontent.com/1595739/117863160-5518a500-b248-11eb-959e-b6ec8aecd2b1.png)
